distributed/tests/test_client.py

```diff
@@ -5255,6 +5255,12 @@ def test_dashboard_link(loop, monkeypatch):
                 assert link in text
 
 
+def test_dashboard_link_inproc(loop, monkeypatch):
```

Comment:
Why monkeypatch here?

If you're comfortable with it, I recommend defaulting to asynchronous tests.  They're faster and easier to debug if anything goes wrong in the future.  Also, if you do end up using synchronous tests then I encourage you to include the `cleanup` fixture.

```python
@pytest.mark.asyncio
async def test_dashboard_link_inproc(cleanup):
    async with Client(processes=False, asynchronous=True) as c: 
        ...

Reply:
Hmm that's some bad copy/paste from the test above. Apologies.

I'll switch it to async.